hack/release.sh

```diff
@@ -29,12 +29,14 @@ readonly SERVING_RELEASE_GCR
 # istio.yaml file to upload
 # We publish our own istio.yaml, so users don't need to use helm"
 readonly ISTIO_YAML=./third_party/istio-1.0-prerelease/istio.yaml
-# Local generated yaml file.
+# Elasticsearch and Kibana yaml file.
+readonly OBSERVABILITY_ELK_YAML=observability_elk.yaml
+# Prometheus and Grafana yaml file.
+readonly OBSERVABILITY_PROMETHEUS_YAML=observability_prom.yaml
+# Prometheus and Grafana yaml file.
+readonly OBSERVABILITY_ZIPKIN_YAML=observability_zipkin.yaml
+# Release with Build + Serving
```

Comment:
I think we may as well split build off while we're doing this, it should be relatively straightforward, but LMK if I can help.

Reply:
We should just do for it what we're doing for Istio